_posts/2019-02-11-lecture-08.md

```diff
@@ -0,0 +1,246 @@
+
+### Kidney Stone Experiment
+Let's revisit the kidney stone treatment example. Let R = Recovery, T = Treatment and S = Stone size, then
+
+$$
+P(R \mid T) = \sum_S P(R \mid S,T)P(S \mid T) 
+$$
+
+$$
+P(R \mid do(T)) = \sum_S P(R \mid S,T)P(S) 
+$$
+
+Note that while intervening, we cut off all incoming edges into the treatment node. So, now S and T are independent.
+
+### Causal Effect Identifiability
+An aspect of a statistical model is identifiable, if it cannot be changed without there also being some change in the distribution of the observed variables. So if we can express a quantity as a function of the distributions of observed variables, then we can say that quantity is identifiable. Thus the causal effect is identifiable if and only if we can represent $$ P(Y \mid do(X)) $$ in terms of the distribution of observed variables. i.e. if
+
+$$ P_{M1}(Y \mid do(X)) = P_{M2}(Y \mid do(X)) $$ for every pair of models M1 and M2 with the dame graph and the same distibution of observed variables(Unobserved / Latent variable distributions can be changed)
+
+Example - If we have just two random variables X and Y(both observed) where X is the cause of Y and there is no confounder.So, $$ P(Y \mid do(X)) = P(Y \mid X) $$ and we know this is identifiable because for changing $$ P(Y \mid X) $$, we would have to change the distribution of P(X,Y).
+
+But, now lets say we have a confounder C, which is the common cause of both X and Y. And this is not observed. Now, we can change the distribution of C such that P(X,Y) is unchanged but we get different $$ P(Y \mid do(X)) $$
+$$ P(Y \mid do(X)) $$ can also be written as $$ P(y \mid \hat{x}) $$
+
+#### Back-Door Criterion 
+A set of variables Z satisfies the **back-door criterion** relative to an ordered pair of variables $$ (X_i, X_j) $$ if
+
+* No node in Z is a descendant of $$ X_i $$
+* Z blocks every path between $$ X_i $$ and $$ X_j $$ that contains an arrow into $$X_i$$ (Hence the name back-door)
+
+If the set of nodes Z follows these two conditions then $$ P (y \mid \hat{x}) = \sum_z P(y \mid x,z)P(z) $$
+Same as we saw earlier in the case of kidney stone treatment. And thus $$ P(y \mid \hat{x}) $$ is now identifiable because of this back-door adjustment.
+
+
+#### Front-Door Criterion
+A set of variables Z satisfies the front-door criterion relative to an ordered pair of variables $$ (X_i, X_j) $$ if
+* Z intercept all directed paths from X to Y (hence the name front-door)
+* There is no back-door path from X to Z 
+* All back-door paths from Z to Y are blocked by X
+If the set of nodes Z follows these two conditions then
+
+$$
+P(y \mid \hat{x}) = \sum_z P(z \mid x)\sum_{x'}P(y \mid x',z)P(x')
+$$
+
+
+
+####  Unification
+
+A sufficient condition under which the causal effect $$ P(y | do(x)) $$ is identifiable is that there exist no bi-directed path between X and any of its children. For more information, this is presented in the paper "A General Identification Condition for Causal Effect".
```

Comment:
This line doesn't seem to render properly:
![image](https://user-images.githubusercontent.com/2617555/53582678-eca0ea00-3b4d-11e9-8ee7-857ac4247320.png)